packages/plugin-flex/src/commands/flex/plugins/upgrade-plugin.ts

```diff
@@ -483,6 +568,22 @@ export default class FlexPluginsUpgradePlugin extends FlexPlugin {
     return semver.coerce(pkg)?.major;
   }
 
+  /**
+   * Returns the flex-ui version from the plugin
+   */
+  get flexUIVersion(): number | undefined {
+    const pkg =
+      this.pkg.dependencies[flexUI] ||
+      this.pkg.devDependencies[flexUI] ||
+      this.pkg.dependencies[flexUI] ||
+      this.pkg.devDependencies[flexUI];
+    if (!pkg) {
+      throw new TwilioCliError(`Package '${flexUI}' was not found`);
+    }
+
+    return semver.coerce(pkg)?.major;
```

Comment:
`return semver.coerce(pkg)?.major || FlexPlugin.DEFAULT_FLEX_UI_VERSION` and define DEFAULT_FLEX_UI_VERSION to be 1